staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -127,9 +131,7 @@ func (t *azureDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume *v1.Vol
 		pv.Spec.PersistentVolumeSource.CSI.FSType = *azureSource.FSType
 		pv.Spec.PersistentVolumeSource.CSI.VolumeAttributes[azureDiskFSType] = *azureSource.FSType
 	}
-	if azureSource.Kind != nil {
-		pv.Spec.PersistentVolumeSource.CSI.VolumeAttributes[azureDiskKind] = string(*azureSource.Kind)
-	}
+	pv.Spec.PersistentVolumeSource.CSI.VolumeAttributes[azureDiskKind] = managed
```

Comment:
what is the behavior if a previously unmanaged inline volume is handled as a managed volume by the CSI driver? Does that break anything?

Reply:
unmanaged volume is not supported in Azure Disk CSI driver, I have add deprecation notice in original PR:

```
 - deprecation notice:
1) `Shared`, `Dedicated` kind is not supported for Azure Disk CSI driver, only `managed` kind is supported in CSI migration.
2) `Shared`, `Dedicated` kind is deprecated in in-tree Azure Disk driver, and will receive error when "CSIMigrationAzureDisk" is turned on
```

Reply:
I think that should not change the functionality for inline volume.  @andyzhangx @msau42 please confirm.

Reply:
for an unmanaged volume if CSI translation is turned on, it would return error for existing unmanaged PV, cannot convert to a CSI volume.

Reply:
ah, any kind other than Managed returns an error above, and API defaulting ensures Kind is always populated